duckduckgo/Android
# Update user agent to fix breakages

Task/Issue URL: https://app.asana.com/0/72649045549333/1176477052804241/f
Tech Design URL: https://app.asana.com/0/361428290920652/1179126436918042/f

**Description**:
Updates user agent to include our application component before the safari component (similar to Chrome) and to allow site specific changes.

**Steps to test this PR**:
Test on real devices as emulators could have a really old WebKit version:
1. Visit `https://www.whatismybrowser.com/detect/what-is-my-user-agent` and check that the user agent reported contains our DuckDuckGo component followed by the Safari component
1. Visit `mijn.ing.nl/login/` in both Mobile and Desktop modes and ensure the login page loads
1. Visit `chase.com` in desktop mode and ensure you can press the "sign in" button and load the login page (this page is really slow sometimes!)
1. Visit `cvs.com` (use a VPN located in the US) in both Desktop and Mobile modes and ensure it loads ok

<img width="485" alt="screenshots" src="https://user-images.githubusercontent.com/1652407/85237801-63d44e80-b421-11ea-8783-08a2d3e5a19d.png">

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
